R/deltaccd.R

```diff
@@ -1,12 +1,12 @@
 #' @importFrom foreach foreach %do% %dopar%
 #' @importFrom doRNG %dorng%
-#' @importFrom data.table data.table :=
+#' @importFrom data.table data.table as.data.table setnames :=
```

Comment:
If we're only using these functions once, I prefer to not import them and instead use the ::.